developer-docs-site/docs/concepts/resources.md

```diff
@@ -7,9 +7,9 @@ id: "resources"
 
 On Aptos, on-chain state is organized into resources and modules. These are then stored within the individual accounts. This is different from other blockchains, such as Ethereum, where each smart contract maintains their own storage space. Additionally, [event handles](./events.md) are defined within resources. See [Accounts](./accounts.md) for more details on accounts.
 
-## Resources vs Objects
+## Resources vs Instances
 
-Resources refer to top-level objects that are stored directly with an account on the blockchain. Both resources and objects are instances of structs. Objects can be resources but can also be individual units of state that are stored inside a resource. An example here is how the APT coin is stored: CoinStore is the resource that contains the APT coin, while the Coin itself is an object:
+Struct instances can either be stored as a resource or within a resource. A resource refers to top-level instance that are stored directly with an account on the blockchain. An example here is how the APT coin is stored: CoinStore is the resource that contains the APT coin, while the Coin itself is an instances:
```

Comment:
Resources are structs with `key` ability, but why is it important to compare to normal structs? It seems pretty intuitive that you can store top-level containers only.

Reply:
After reading the section, I see what is meant! I think we mixing multiple things here. I think a text like this is way clearer?

```
Move modules contain struct definitions, which can be created in other modules or stored on-chain.
Structs that are stored on-chain are called resources.
To be a resource struct definition must have a `key` ability.
```
or along these lines?

Reply:
Yeah, that's a better work on my tweaks. I'll try to incorporate it.